content/getting-started/setup/upgrades/index.md

```diff
@@ -8,4 +8,63 @@ dnnversion: 09.04.03
 # DNN Platform Upgrades
 Following are the basic steps for upgrading **DNN Platform**.
 
-Please click the **Improve this Doc** button above to help us improve this page.
\ No newline at end of file
+> [!NOTE]
+> Every upgrade scenario is unique in some shape, form or fashion. This document is by no means intended to address every scenario. Some upgrades can be quite complex and it is virtually impossible to cover every scenario. This document is intended to focus only on the upgrade basics.
+
+## Step 1 - Prepare
+Before starting the upgrade process, it is recommended to have a very clear understanding of all upgrade requirements and expectations. Here are some tips to help prepare.
+
+* Know the current version of DNN Platform.
+
+* Ensure **AutoUpgrade** setting is "False". This setting can be found within **AppSettings** section of the `web.config` file in the root. Setting this to "False" will prevent the upgrade process from triggering by normal site visitors. 
+
+* Know the desired version of DNN Platform and download the appropriate "Upgrade" ZIP package from [the official repository on GitHub](https://github.com/dnnsoftware/DNN.Platform/releases) (which can be found in the **Assets** section for the desired version release).
```

Comment:
Perhaps add an example of what the filename of the upgrade zip file looks like? Not required but may be helpful.

Reply:
I suppose we could, but the file naming convention has changed over the years.

Reply:
Yeah, I'm aware of the different conventions. Not a big deal, but I'm thinking of it from a newbie perspective. Sometimes it's better to be explicit and not assume they know what "Upgrade" file is. I think if we said: While the naming convention has changed over the years, the file will contain the word "Upgrade" in it. For instance, for version 9.4.4, the filename is  DNN_Platform_9.4.4_Upgrade.zip" . Again, just a thought.

Reply:
I am including this change in a new PR.